applications/matter_weather_station/README.rst

```diff
@@ -190,15 +190,20 @@ You can generate new factory data set when building for the target board by invo
 .. parsed-literal::
    :class: highlight
 
-   west build -b thingy53_nrf5340_cpuapp -- -DCONF_FILE=prj_factory_data.conf -DOVERLAY_CONFIG="../../overlay-factory_data_build.conf"
+   west build -b thingy53_nrf5340_cpuapp -- -DOVERLAY_CONFIG=overlay-factory_data.conf
 
+This command builds the application with default certificates.
 After building the target, the generated :file:`factory_data.hex` file will be merged with the application target HEX file, so you can use the regular command to flash it to the device:
 
 .. parsed-literal::
    :class: highlight
 
    west flash --erase
 
+If you want to use Vendor ID, Product ID or other data that is not reserved for tests, you need custom certificates.
```

Comment:
I will change name from `custom certificates` to `custom test certificates` or `not final custom certificates`
Basically the device maker should use their of external PKI infrastructure to generate final PKI certificates for their production